test/PlayerActions.test.js

```diff
@@ -6,7 +6,7 @@ const { GameState } = require('../entities/GameState');
 const { createPlayer } = require('./testutils');
 const PlayerActions = require('../entities/PlayerActions');
 
-describe('PlayerActions', function () {
+describe('PlayerActions', () => {
```

Comment:
_*Note*_: This was probably my doing - in Mocha tests, if you use `function`s then there is a test context bound to `this`. I was using it for some dynamic tests back when this started, and it is an artefact of that.
I'm pointing this out now because we might want to revisit it later -- I had _some reason_ (I don't remember now) for doing this and getting the test context bound; an investigation might yield some fruitful knowledge for us